ERCS/erc-operation-receipts.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
```suggestion
---
eip: 7655
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.